modules/LocalStorage.js

```diff
@@ -0,0 +1,14 @@
+const storageKey = 'awesomeBooks';
+function loadDataFromLocalStorage() {
+  const dataLoaded = JSON.parse(localStorage.getItem(storageKey));
+  if (dataLoaded == null) {
+    return [];
+  }
+  return dataLoaded;
+}
+
+function saveDataToLocalStorage(arrBooks) {
+  localStorage.setItem(storageKey, JSON.stringify(arrBooks));
+}
+
```

Comment:
- [ ]Again here kindly use arrow format for functions here, as this is the given requirement